partner_tz/tools/tz_utils.py

```diff
@@ -0,0 +1,58 @@
+# Copyright 2020 Camptocamp SA
```

Comment:
The functions presented here are very thin wrappers around the original functions from pytz. I think this is of questionable value. Better use the original pytz functions that are generally available and not depending on having this particular module installed.

Reply:
Thx for your review @NL66278 
The value of a function is quite subjective in any case, but to me they make tz conversions more explicit and avoid repeats. Moreover they are only tools that won't change anything when someone installs this module.
So I'm sorry but I don't see why having this file here, and using it in other modules having this module as a dependency would be a problem...